add-on/_locales/en/messages.json

```
@@ -726,5 +726,29 @@
   "page_landingWelcome_projects_title": {
     "message": "Related Projects",
     "description": "Projects section title (page_landingWelcome_projects_title)"
+  },
+  "recovery_page_title" : {
+    "message": "Node Offline | IPFS Companion",
+    "description": "Title of the recovery page (recovery_page_title)"
+  },
+  "recovery_page_header" : {
+    "message": "Node is Offline",
+    "description": "Main header on the recovery screen (recovery_page_header)"
+  },
+  "recovery_page_sub_header": {
+    "message": "IPFS Companion is unable to connect to this IPFS node.",
+    "description": "Sub-Header on the recovery screen (recovery_page_sub_header)"
+  },
+  "recovery_page_message": {
+    "message": "Your configured node seems to be offline. Would you like to connect to a public gateway instead?",
+    "description": "Message on the recovery screen (recovery_page_message)"
+  },
+  "recovery_page_button": {
+    "message": "Continue to a public gateway",
```

My English is not the best but I think its `the` since we show specific URL, and we have specific public gateway in settings.

```suggestion
    "message": "Continue to the public gateway",
```